README.md

```diff
@@ -17,3 +17,120 @@ with Zipkin-compatible data model.
+
+### Dropwizard: ###
+
+Jaeger’s dropwizard jar exposes a configuration object `com.uber.jaeger.dropwizard.Configuration`  If your service reads in its configuration through a pojo object then you can use this Configuration class to read the yaml format specified above.
+The following is an example of adding a client filter to a JAX-RS based framework that creates a dropwizard/jersey client.
+
+Client instrumentation example:
+```
+import com.uber.jaeger.dropwizard.TracingUtils;
+
+// JaegerConfig gets initialized by whichever dropwizard configuration reader you are using.
+com.uber.jaeger.dropwizard.Configuration jaegerConfig;
+
+final Client client = new JerseyClientBuilder(env)
+    /* Arbitrary configuration code … */
+    .register(TracingUtils.clientFilter(jaegerConfig);
+```
+
+Your server configuration will probably not look exactly like the one below.  However JAX-RS server configurations always expose a `register` function that can be used to register a server filter like below.
+Server instrumentation example:
+```
+Final ResourceConfig rc = new ResourceConfig()
+    /* Arbitrary configuration code… */
+    .register(TracingUtils.serverFilter(jaegerConfig));
+
+return GrizzlyHttpServerFactory.createHttpServer(URI.create(BASE_URI), rc);
+```
+
+### Metrics ###
+In order for Jaeger to set metrics properly it needs to be configured with a metric registry.  
+```
+import com.uber.jaeger.dropwizard.TracingUtils;
+
+// Dropwizard's metrics registry class
+MetricsRegistry registry = new MetricsRegistry();
+
+// JaegerConfig gets initialized by whichever dropwizard configuration reader you are using.
+com.uber.jaeger.dropwizard.Configuration jaegerConfig;
+
+jaegerConfig.setMetricRegistry(registry);
+
+final Client client = new JerseyClientBuilder(env)
+    /* Arbitrary configuration code … */
+    .register(TracingUtils.clientFilter(jaegerConfig);
+```
+
+## Thread Pooling: ##
+Because Jaeger uses thread local storage in order to propagate information without changing your code.  The creation of new thread, or the reuse of threads needs to be wrapped in our instrumented tools in order to preserve the structure of a Trace in the UI.
+
+An executor service needs to be wrapped in our TracedExecutorService. 
+
+Executor Service
+```
+ExecutorService instrumentedExecutorService = TracingUtils.tracedExecutor(wrappedExecutorService);
+```
+
+## Testing ##
+At times you may need to force spans to be sampled in order to test your instrumentation, or diagnose a problem in production.  The following format allows you to specify the sampler to use:
```

Comment:
First sentence completely misleading. Forcing sampling and configuring traced with 100% sampling are different things.